app/authentication/authenticator.py

```diff
@@ -22,27 +24,31 @@
 
 
 @login_manager.user_loader
-def user_loader(user_id):
+def user_loader(user_id: str) -> Union[str, None]:
     logger.debug("loading user", user_id=user_id)
     return load_user()
 
 
 @login_manager.request_loader
-def request_load_user(request):  # pylint: disable=unused-argument
+def request_load_user(
+    request: Any,
```

Comment:
We need to minimise the use of `Any`. It should not be used unless it can't be avoided. Here this is a flask request so `Request` from flask.